### PR TITLE
Clue dice

### PR DESCRIPTION
**Pull request by dastels**
New PR with errant M4eyes file removed.
Comments from previous PR incorporated.

### CODE REVIEW

File: CLUE_Dice_Roller/clue_dice_roller.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+display
+mode = SELECTING
+update_display(number_of_dice, side_selection)
+
+while True:
+    button_a.update()
+    button_b.update()
+
+    if mode == SELECTING:
+        if button_a.rose:
```
Comment (ladyada) [inline CLUE_Dice_Roller/clue_dice_roller.py]: i think people expect it to change on falling (pressed) not rising?

Comment (dastels) [reply CLUE_Dice_Roller/clue_dice_roller.py]: The clue button properties return True when pressed.  As does the CPB buttons.  Both those surprised me for just that reason.

**Comment by ladyada** [comment]
ok nice! plz lint :)

**Comment by dastels** [comment]
Yeah.. linting... it's been a while :)

**Comment by dglaude** [comment]
I believe d10 is missing from your code, but is advertised in the comment:

SIDES = [4, 6, 8, 12, 20, 100] =>SIDES = [4, 6, 8, 10, 12, 20, 100]

**Comment by dastels** [comment]
Doah.. d10 is definitely missing.

**Pull request merged**